www/include/configuration/configObject/host_categories/DB-Func.php

```diff
@@ -85,95 +86,174 @@ function shouldNotBeEqTo0($value)
     }
 }
 
-function enableHostCategoriesInDB($hc_id = null, $hc_arr = array())
+function enableHostCategoriesInDB(int $hcId = null, $hc_arr = [])
```

Comment:
@jeremyjaouen , you forgot the $hc_arr when you remaned in camelCase the variables